webapp/src/constants/constants.ts

```diff
@@ -0,0 +1,5 @@
+export default {
+    MEETING_ID: 'meetingID',
+    JAAS_EPHEMERAL_MESSAGE: 'You cannot open a Jaas meeting when the selected server type is Jitsi. Please contact your system administrator.',
```

Comment:
Divide this message. The first line is common in both the messages so you can create a variable for that.